src/components/Companies/CompaniesForm/index.js

```diff
@@ -153,7 +155,9 @@ function getFieldComponent({
 }
 
 export const getDefaultValuesFromDataObject = (formObject, editMode = false) => {
-  const defaultValues = {};
+  const defaultValues = {
+    signedContract: Boolean(formObject.id),
```

Comment:
Why is signedContract formobjectId?

Reply:
Because in order to create object with ID you need to submit signed form. Therefore, any form with ID is also signed